tsstats.sample.ini

```diff
@@ -33,7 +33,10 @@ owners: uid1
 commandprefix: !
 
 ;uid of clients you don't want to be counted, separate multiple uids by commas (no spaces)
-ignore : XXXXXXXXXXXXXXXXXXX
+igclient: XXXXXXXXXXXXXXXXXXX
+
+;cid of clients you don't want to be counted, separate multiple cids by commas (no spaces)
+igchannel: X,Y,Z
```

Comment:
`ignoredchannels` might be a better name (also update the comment above it: cid of _channels_)